server/models/connections.go

```diff
@@ -39,7 +39,7 @@ type Connection struct {
 
 // swagger:response ConnectionPage
 type ConnectionPage struct {
-	Connection []Connection `json:"connection"`
+	Connections []Connection `json:"connections"`
```

Comment:
@abdullah1308 This will also require changes in UI, right?

Reply:
@MUzairS15 These endpoints aren't being used anywhere.